### PR TITLE
Star type probability

### PR DESCRIPTION
**Pull request by lea-hagen**
Along with #493, this recreates the content in #468.  This PR has just the code to create star type probabilities from the saved lnPs.  However, since we've decided to try going the route of 2D PDFs rather than using lnPs (#472), I'll need to edit this.

### CODE REVIEW

**Comment by lea-hagen** [comment]
The code/docs are complete.  Feedback (small or large) appreciated, particularly on the idea of making some sort of ascii input list of cuts (as @karllark suggested in #468).

**Comment by karllark** [comment]
No longer WIP?  Please remove WIP from the title of this PR.

File: beast/tools/star_type_probability.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+
+def star_type_probability(
+    pdf1d_files,
+    pdf2d_files,
+    output_filebase=None,
+    ext_O_star_params=None,
+    dusty_agb_params=None,
+):
+    """
```
Comment (karllark) [inline beast/tools/star_type_probability.py]: Need a short description of what this function is doing.

Comment (karllark) [reply beast/tools/star_type_probability.py]: Standard part of docstrings.

Comment (lea-hagen) [reply beast/tools/star_type_probability.py]: Thanks for catching that!

**Pull request merged**